src/app/view/feed/idea-feed/idea-feed.component.html

```diff
@@ -28,7 +28,6 @@
               <mat-icon class="idea-card__icon">arrow_drop_up</mat-icon>
               <div class="idea-card__stat">({{ item.upvotes }})</div>
               <mat-icon class="idea-card__icon" (click)="openDialog(item)">open_in_new</mat-icon>
-              <mat-icon class="idea-card__icon" (click)="deleteidea(item.id)">delete</mat-icon>
```

Comment:
Also remove the `deleteIdea` function from `idea-feed.component.ts`

Reply:
Removed that code from idea-feed.component.ts